test/common.js

```diff
@@ -445,3 +446,11 @@ exports.fileExists = function(pathname) {
     return false;
   }
 };
+
+exports.busyLoop = function busyLoop(time) {
```

Comment:
Yes, if you could move it to a function in the bottom of the test file that would be better for now.